app/views/events/show.html.erb

```diff
@@ -1,55 +1,67 @@
 <p id="notice"><%= notice %></p>
 
 <p>
-  <strong>Title:</strong>
-  <%= @event.title %>
+  <strong><%=I18n.t("events.title") %>:</strong>
+  <%= @event.title %> <span class="badge badge-primary ml-1"><%= @event.eventType %></span>
 </p>
 
 <p>
-  <strong>Description:</strong>
+  <strong><%=I18n.t("events.description") %>:</strong>
   <%= @event.description %>
 </p>
 
 <p>
-  <strong>Startdate:</strong>
-  <%= @event.startDate %>
+  <strong><%=I18n.t("events.when") %>:</strong>
+  <%= get_when_text(@event) %>
 </p>
 
 <p>
-  <strong>Starttime:</strong>
-  <%= @event.startTime %>
+  <strong><%=I18n.t("events.where") %>:</strong>
+  <%= @event.location %>
 </p>
 
 <p>
-  <strong>Enddate:</strong>
-  <%= @event.endDate %>
+  <strong><%=I18n.t("events.published") %>:</strong>
+  <%= @event.published %>
 </p>
-
-<p>
-  <strong>Endtime:</strong>
-  <%= @event.endTime %>
+<% non_message, not_delivered, delivered, not_responded, not_attending, attending =
+       get_users_state_counts(@event) %>
+<% if current_user && !current_user.admin && @event.eventType == "event"%>
```

Comment:
The event type should be an enum, ideally we would want to do @event.eventType.event